contents/dataVis.md

```diff
@@ -0,0 +1,390 @@
+# Data Visualisation {#sec:dataVis}
+
+Data visualisation it is a vital part of almost any data analysis process.
+Here, in this chapter we will consider several libraries that can be used in Julia, namely Plots, Makie and AlbegraOfGraphics specially usefull for tabular data. 
+
+- Overview of the JuliaPlots ecosystem
+
+## Plots.jl {#sec:plots}
+Plots is .... 
+
+The default backend is GR. 
+
+```
+using Plots
+```
+
+### Plots attributes {#sec:attributes}
+
+### Colors and Palettes {#sec:colors}
+Sequential, diverging, categorical. 
+
+### Layouts {#sec:layouts}
+
+- Overview on several ways to do layouts
+- the layout argument, also cover the grid
+- the @layout macro
+- specific measures with the Plots.PlotMeasures submodule (additionally a footnote to say that we have it in StatsPlots.PlotMeasures also)
+- adding subplots incremententally. Define p1, p2, p3; then do a plot(p1, p2, p3; layout=l)
+
+### Animations {#sec:animations}
+Doing an animation with Plots is quite easy. 
+
+### Themes {#sec:themes}
+
+### Other backends {#sec:backends}
+Plotly, PGFPlotsX, PyPlot?
+
+## StatsPlots.jl {#sec:statsplots}
+
+1. A brief intro and show that is is really just "syntactic sugar" for Plots.jl. Also note that works on all plots.jl stuff:
+
+- Backends
+- Attributes
+- Colours
+- Layouts
+- Animations
+- Themes
+2. Works on Types of Distributions and DataFrames, but we will only focus on DataFrames
+
+3. the @df macro and the Symbols (:col1, :col2) instead of other input data (vectors, arrays, tuples etc.)
+
+4. Recipes (I would cover almost all of them, except for andrewsplot MDS plot, Dendograms, QQ-Plot etc., since they are specific for modeling or simulation):
+
+- histogram, histogram2d and ea_histogram
+- groupedhist and groupedbar
+- boxplot
+- dotplot
+- violin
+- marginalhist, marginalkde and marginalscatter
+- corrplot and cornerplot
+
+## Makie.jl {#sec:makie}
+
+(WIP) ideas for a better openinig line? 
```

Comment:
We could say that we think it is the prettiest plotting library (currently) in Julia?